recipes/fast-pytorch-kmeans/LICENSE.txt

```diff
@@ -0,0 +1,17 @@
+MIT License
+Copyright (c) 2018 YOUR NAME
```

Comment:
This is in upstream, but it is wrong. Can you please open an issue upstream and request them to add the correct name.